internal/repositories/vehicles_test.go

```diff
@@ -785,37 +785,37 @@ func (o *AccessibleVehiclesRepoTestSuite) Test_GetAccessibleVehicles_Pagination_
 
 	last := 2
 	after := "NA=="
-	startCrsr := "Mw=="
+	startCrsr := "MQ=="
```

Comment:
If this comes from fixing pagination, can you cherry-pick all of this to main?

Reply:
Sure